docs: update "Upgrade Path Enforcement and Downgrade Prevention" (#1113)
longhorn/website: content/docs/1.6.0/deploy/upgrade/_index.md

# Upgrade Path Enforcement and Downgrade Prevention

> Starting with v1.5.0, Longhorn only allows upgrades from from one minor version to the next. For example, upgrading from 1.5.x to 1.6.x is allowed, but skipping minor versions (e.g., from 1.4.x to 1.6.x) is not supported. When you attempt to upgrade from an unsupported version or skip a minor version, the operation automatically fails. However, you can revert to the previously installed version without any service interruption or downtime.

Reviewer: ```suggestion
Starting with v1.5.0, Longhorn only supports upgrades from one minor version to the next. For example, upgrading from 1.5.x to 1.6.x is supported, but skipping versions (e.g., from 1.4.x to 1.6.x) is not. If you attempt to upgrade from an unsupported version or skip a minor version, the operation will fail automatically. However, you can revert to the previously installed version without service interruption or downtime.
```
